.github/workflows/test.yml

```diff
@@ -70,22 +70,14 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
       - name: Assemble
-        run: ./gradlew assemble
-      - name: Validate Docker Configurations
-        # `docker-compose` and `docker compose` have different "leniencies"
-        run: |
-          find . -name docker-compose\*.yml | while read -r file; do \
-            echo "$file"; \
-            docker-compose -f "$file" config -q && \
-            docker compose -f "$file" config -q; \
-          done
+        run: ./gradlew assemble --parallel
       - name: Start Services
         run: |
           docker compose -f ./e2e-tests/docker-compose-test.yml up -d --build
           docker images
           docker ps -a
-      - name: Sleep for 15 Seconds
-        run: sleep 15s
+      - name: Sleep for 30 Seconds
+        run: sleep 30s
```

Comment:
Why was this sleep increased?

Reply:
I experienced a few flaky tests where I think aerie wasn't fully initialized before e2e test started running, so I increased this sleep since we saved several minutes elsewhere.